internal/manifest/list.go

```diff
@@ -0,0 +1,24 @@
+package manifest
+
+import (
+	publicManifest "github.com/containers/image/v5/manifest"
+)
+
+// List is an interface for parsing, modifying lists of image manifests.
+// Callers can either use this abstract interface without understanding the details of the formats,
+// or instantiate a specific implementation (e.g. manifest.OCI1Index) and access the public members
+// directly.
+type List interface {
+	publicManifest.List
+}
+
+// ListFromBlob parses a list of manifests.
+func ListFromBlob(manifest []byte, manifestMIMEType string) (List, error) {
+	return publicManifest.ListFromBlob(manifest, manifestMIMEType)
+}
+
+// ConvertListToMIMEType converts the passed-in manifest list to a manifest
+// list of the specified type.
+func ConvertListToMIMEType(list List, manifestMIMEType string) (List, error) {
```

Comment:
There are no callers of this trivial wrapper, so I think that can stay in the public package.